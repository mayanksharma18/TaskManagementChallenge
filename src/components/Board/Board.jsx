import React from "react";
import CreateBoard from "./CreateBoard/CreateBoard";
import BoardsList from "./BoardsList/BoardsList";

const Board = () => {
  return (
    <div style={{ display: "flex", flexDirection: "column", marginTop: "20px" }}>
      <CreateBoard />
      <div style={{ marginTop: "80px" }}>
        <BoardsList />
      </div>
    </div>
  );
};

export default Board;
