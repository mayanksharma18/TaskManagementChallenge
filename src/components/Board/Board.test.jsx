import React from "react";
import { render } from "@testing-library/react";
import Board from "./Board";
jest.mock("./CreateBoard/CreateBoard", () => () => (
  <div data-testid="mock-board">Create Board</div>
));
jest.mock("./BoardsList/BoardsList", () => () => (
    <div data-testid="mock-board">Board List</div>
  ));

describe("Board", () => {
  it("renders correctly", () => {
    const { container } = render(<Board />);
    expect(container).toMatchSnapshot();
  });
});
