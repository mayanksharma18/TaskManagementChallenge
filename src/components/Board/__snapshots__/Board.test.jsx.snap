// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board renders correctly 1`] = `
<div>
  <div
    style="display: flex; flex-direction: column; margin-top: 20px;"
  >
    <div
      data-testid="mock-board"
    >
      Create Board
    </div>
    <div
      style="margin-top: 80px;"
    >
      <div
        data-testid="mock-board"
      >
        Board List
      </div>
    </div>
  </div>
</div>
`;
